tests/unit/tracking.tcl

```diff
@@ -344,7 +354,6 @@ start_server {tags {"tracking network"}} {
     }
 
     test {Tracking gets notification on tracking table key eviction} {
-        $rd_redirection HELLO 2
```

Comment:
can you explain why you removed this?

Reply:
In test {Able to redirect to a RESP3 client}, I make`$rd_redirection` RESP2. For the principle, change and recovery in the same test.
Besides,  when execute `$rd_redirection HELLO 2`, it doesn't read the reply.